.github/workflows/pr.yaml

```diff
@@ -73,7 +73,7 @@ jobs:
       - name: Set up Node.js 16.x
         uses: actions/setup-node@v1
         with:
-          node-version: 16.x
+          node-version: 16.18.1
```

Comment:
any reason behind this?

Reply:
It will be the best to everyone to use same version to avoid any unecessary issue may cause. I have tested much with this version, and it's pretty new so it's will be the best to choice it.

Reply:
FYI we already discuss about this before: https://team-kyber.slack.com/archives/C033XMZUMEX/p1662691792633329?thread_ts=1662689846.197979&cid=C033XMZUMEX